charts/airbyte/values.yaml

```diff
@@ -549,3 +549,9 @@ logs:
   gcs:
     bucket: ""
     credentials: ""
+
+minio:
+  accessKey:
+    password: minio
+  secretKey:
+    password: minio123
```

Comment:
where are you planning to use this? The minio endpoint use `externalMinio` at the moment.

Reply:
If I do a recursive search in the chart folder, I never find `.Values.minio` but only `.Values.logs.minio` so I don't understand how this is used ?
Are those values shared with the minio chart too ? If so, we cannot mutualize ak/sk with S3 and thus must refacto this.

Reply:
If Values are used by the minio chart too it's normal indeed. I'll try to propose a PR to fix this without repeating later.

Reply:
Sounds good to me 👍